src/training/lstmtraining.cpp

```diff
@@ -78,6 +78,8 @@ int main(int argc, char **argv) {
     tprintf("Must provide a --model_output!\n");
     return 1;
   }
+  if (CreateDirIfNotExists(FLAGS_model_output.c_str(), "model_path"))
```

Comment:
`FLAGS_model_output` is the basename for output models, not a directory name.

Reply:
so GetDirectory should be used first...